pkg/datapath/linux/probes/probes.go

```diff
@@ -201,10 +203,25 @@ func (p *ProbeManager) SystemKernelHz() (int, error) {
 // warnings when optional parameters are not enabled.
 func (p *ProbeManager) SystemConfigProbes() error {
 	config := p.features.SystemConfig
-	// Required
-	if !config.ConfigBpf.Enabled() {
-		return ErrKernelConfigNotFound
+
+	// Check Kernel Config is available or not.
+	// We are replicating BPFTools logic here to check if kernel config is available
+	// https://elixir.bootlin.com/linux/latest/source/tools/bpf/bpftool/feature.c#L390
```

Comment:
```suggestion
	// https://elixir.bootlin.com/linux/v5.7/source/tools/bpf/bpftool/feature.c#L390
```
Line numbers (or code) are susceptible to change, please specify a version.